core/src/main/java/io/github/thibaultbee/streampack/internal/encoders/MultiVideoEncoderTargetInfo.kt

```diff
@@ -0,0 +1,286 @@
+package io.github.thibaultbee.streampack.internal.encoders
+
+import android.graphics.SurfaceTexture
+import android.media.MediaCodec
+import android.media.MediaCodecInfo
+import android.media.MediaFormat
+import android.os.Build
+import android.os.Bundle
+import android.os.Handler
+import android.os.HandlerThread
+import android.util.Size
+import android.view.Surface
+import io.github.thibaultbee.streampack.data.VideoConfig
+import io.github.thibaultbee.streampack.error.StreamPackError
+import io.github.thibaultbee.streampack.internal.data.Frame
+import io.github.thibaultbee.streampack.internal.gl.EglDisplayContext
+import io.github.thibaultbee.streampack.internal.gl.EglWindowSurface
+import io.github.thibaultbee.streampack.internal.gl.FullFrameRect
+import io.github.thibaultbee.streampack.internal.gl.Texture2DProgram
+import io.github.thibaultbee.streampack.internal.orientation.ISourceOrientationProvider
+import io.github.thibaultbee.streampack.internal.utils.av.video.DynamicRangeProfile
+import io.github.thibaultbee.streampack.logger.Logger
+import io.github.thibaultbee.streampack.streamers.interfaces.settings.IVideoSettings
+import java.util.concurrent.ExecutorService
+
+data class MultiVideoEncoderTargetInfo(
```

Comment:
obviously this code is pretty similar to alot of VideoMediaCodecEncoder.CodecSurface. Perhaps someday the two might be one.